### PR TITLE
org.gtk.Gtk3theme.Catppuccin-teal

### PR DESCRIPTION
**Pull request by Scrumplex**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/catppuccin/gtk/issues/19
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

There are multiple variants of this theme. I would create the other PRs once this gets added to flathub.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.gtk.Gtk3theme.Catppuccin-teal.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        dest-filename: "theme.zip"
+        sha256: ba8ac119f9f6fb7bcab05794232ed424ba5e1f80cf11f2f4d04ebe744e3d91a9
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 org.gtk.Gtk3theme.Catppuccin-teal.appdata.xml -t ${FLATPAK_DEST}/share/appdata
+      - appstream-compose --basename=org.gtk.Gtk3theme.Catppuccin-teal --prefix=${FLATPAK_DEST} --origin=flatpak org.gtk.Gtk3theme.Catppuccin-teal
+    sources:
+      - type: file
+        path: org.gtk.Gtk3theme.Catppuccin-teal.appdata.xml
```
Comment (hfiguiere) [inline org.gtk.Gtk3theme.Catppuccin-teal.yml:29]:
```suggestion
  - name: Catppuccin-teal
    buildsystem: simple
    build-commands:
      - install -dm755 /usr/share/runtime/share/themes
      - unzip theme.zip Catppuccin-teal/gtk-3.0/* Catppuccin-teal/index.theme -d /usr/share/runtime/share/themes
      - install -Dm644 org.gtk.Gtk3theme.Catppuccin-teal.appdata.xml -t ${FLATPAK_DEST}/share/appdata
      - appstream-compose --basename=org.gtk.Gtk3theme.Catppuccin-teal --prefix=${FLATPAK_DEST} --origin=flatpak org.gtk.Gtk3theme.Catppuccin-teal
    sources:
      - type: file
        url: https://github.com/catppuccin/gtk/releases/download/update_23_02_2022/Catppuccin-teal.zip
        dest-filename: "theme.zip"
        sha256: ba8ac119f9f6fb7bcab05794232ed424ba5e1f80cf11f2f4d04ebe744e3d91a9
      - type: file
        path: org.gtk.Gtk3theme.Catppuccin-teal.appdata.xml
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.gtk.Gtk3theme.Catppuccin-teal

**Comment by Scrumplex** [comment]
bot, build org.gtk.Gtk3theme.Catppuccin-teal

**Pull request closed without merging**